students/uss_tomasz/lesson_05_lists/5_2/character_picture_grid.py

```diff
@@ -0,0 +1,26 @@
+def make_grid_transpose(grid_from_user):
+    column_quantity = len(grid_from_user[0])
+    return [[row[i] for row in grid_from_user] for i in range(column_quantity)]
```

Comment:
looks compact and clever, but it's hard to follow what's going on here
for production code it should rather be changed to more verbose solution, but here we can keep it as it is